package.json

```diff
@@ -41,6 +44,7 @@
     "@testing-library/react": "8.0.1",
     "@testing-library/react-hooks": "1.1.0",
     "check-dependencies": "1.1.0",
+    "concurrently": "4.1.1",
```

Comment:
why we need that?

Reply:
I dont like as I dont see how it solves the problem.

Reply:
It's a small tool that allows to run two npm tasks in parallel. Currently if you want to run e2e tests, you need to manually trigger two tasks: `npm start` and after dev server starts `npm run e2e:dev`.  Now single npm task runs web server and waits until it initialise and _after_ that it runs E2E tests. This feature was requested here: https://github.com/BuildIt-Poland/daily-routine/pull/111#discussion_r310983170 by @blurbyte

Reply:
ok, I think I get it, on each dependant task you add the waiting, so you can safely run concurently, I guess its fine, as long as you are certain concurently yields rc of last process (and this could be done with bash also, I suppose)

Reply:
perhaps `&` would suffice?

Reply:
>perhaps & would suffice?

I explained how it works here https://github.com/BuildIt-Poland/daily-routine/pull/111#discussion_r311465237

> I guess its fine, as long as you are certain concurently yields rc of last process (and this could be done with bash also, I suppose)

Of course it could be done in the bash by custom script, but it's only a small dev dependency used very widely and create for such purposes. Running couple task in parallel is very common and this simple fix this problem. Another advantage is cross-platform solution so it's gonna work under Windows as well.

You can look at the source code of this here: https://github.com/kimmobrunfeldt/concurrently

Reply:
ok, im fine with this, but im not fine with argumentation that its **just** a small dependency.... this why I usually dislike npm stuff, because jsland tries to rewrite everylittle tool and utility... and suddenly boom, to run a simple static website you have to download 300mb of dependencies...
and this is not a random example, i did check "our" node_modules

as for windows, i think recommend course of actions is to use WSL and ignore the host environment

Reply:
>ok, im fine with this, but im not fine with argumentation that its just a small dependency.... 

As we discuss it via Slack, I don't mind replacing `concurrently` dependency with `&` 😎 even it's a cross-platform ready tool and using `&` may not work correctly on different OS (e.g. Windows). More about it [here](https://github.com/mysticatea/npm-run-all). 

Another drawback of using `&`  in our project is that after running ```npm start & npm run cypress``` from the terminal the webback dev server running at `http://localhost:3000/`  will remains working in the background and needs to be manually searched and killed (CMD/CTRL+C won't work). Otherwise process will block port 3000 - good to keep that in mind, since E2E tests works  are configured to work only on this port.

Anyway, I removed `concurrently`  in this commit  👉https://github.com/BuildIt-Poland/daily-routine/pull/111/commits/d799a7e87f7809c30b840d8390b27478166f82dc

>and suddenly boom, to run a simple static website you have to download 300mb of dependencies... and this is not a random example, i did check "our" node_modules

I won't argue as I know exactly what you mean. Not a fun of using a lib, but as always... _it depends..._ 😀

Reply:
ok, one last thing :P
to make it more usable in the case you just described -> I would switch the commands.
So that `npm run cypress & npm start`... this would allow for ctrl+c to abort testing, cypress is expected to end at some point so its less of a bother.

Reply:
https://youtu.be/15HTd4Um1m4?t=10

Reply:
> ok, one last thing :P
> to make it more usable in the case you just described -> I would switch the commands.
> So that `npm run cypress & npm start`... this would allow for ctrl+c to abort testing, cypress is expected to end at some point so its less of a bother.

Good catch! Done in https://github.com/BuildIt-Poland/daily-routine/pull/111/commits/1f174f6c8015ea1f7a52d3d28f2cccffb737c696